x-pack/plugins/infra/public/components/asset_details/components/top_processes_tooltip.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+const DOCUMENTATION_LINK =
+  'https://www.elastic.co/guide/en/observability/current/view-infrastructure-metrics.html';
+const SYSTEM_INTEGRATION_DOCS_LINK = 'https://docs.elastic.co/en/integrations/system';
+
+export const TopProcessesTooltipContent = React.memo(() => {
+  const onClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    e.stopPropagation();
```

Comment:
Could you clarify please why this is needed?

Reply:
It was added initially as a bug fix because of the table sorting issue with the tooltip there https://github.com/elastic/kibana/pull/161565 but in this case, I don't think we need it -  removed 👍